src/modules/upload_scores.js

```diff
@@ -0,0 +1,26 @@
+import {
+  baseUrl, games, id, scores,
+} from './api.js';
+import recentScores from './recent_scores.js';
+import loadScores from './load_scores.js';
+
+const scoresContainer = document.querySelector('.scores-list');
+const path = `${baseUrl}${games}${id}${scores}`;
+async function uploadScore(user, score) {
+  fetch(path, {
+    method: 'POST',
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json())
+    .then(() => {
+      scoresContainer.innerHTML = '';
+      loadScores(scoresContainer, recentScores);
+    });
```

Comment:
- [ ] The functionality of your website is working perfectly but as part of the requirements, we need you to use `async` `await` syntax rather than the `.then` style of using async functions. 

something like this 👇
```suggestion
  const response = await fetch(path, {
    method: 'POST',
    body: JSON.stringify({
      user,
      score,
    }),
    headers: {
      'Content-type': 'application/json; charset=UTF-8',
    },
  });
  await response.json();
  scoresContainer.innerHTML = '';
  loadScores(scoresContainer, recentScores);
```